### PR TITLE
`azurerm_spring_cloud_configuration_service` - support for the `ca_certificate_id` property

### PR DESCRIPTION
**Pull request by ms-henglu**

### CODE REVIEW

File: website/docs/r/spring_cloud_configuration_service.html.markdown [hunk 1]
```diff
@@ -73,6 +73,8 @@ A `repository` block supports the following:
 
 * `uri` - (Required) Specifies the URI of the repository.
 
+* `ca_certificate_id` - (Optional) Specifies the ID of the CA Spring Cloud Certificate for https URL of Git repository.
```
Comment (tombuildsstuff) [inline website/docs/r/spring_cloud_configuration_service.html.markdown:76]: Presumably, this is:

```suggestion
* `ca_certificate_id` - (Optional) Specifies the ID of the Certificate Authority used when retrieving the Git Repository via HTTPS.
```

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
One comment around the documentation for this field, but this otherwise LGTM 👍

**Comment by tombuildsstuff** [comment]
@ms-henglu the tests are repeatedly returning an error:

```
 Error: waiting for creation/update of Spring Cloud Configuration Service: (Configuration Service Name "default" / Spring Name "acctest-sc-230807045734471977" / Resource Group "acctestRG-spring-230807045734471977"): Code="CreateOrUpdateAcsFailed" Message="CreateOrUpdateAcsFailed"
```

Is additional configuration needed for this?

**Comment by ms-henglu** [comment]
Hi @tombuildsstuff ,

Thanks for the heads up! I've updated this PR, and the tests are passing:
image

**Review by stephybun — APPROVED** [review]
Thanks @ms-henglu LGTM 🌟

**Pull request merged**